drivers/flash/Kconfig.mcux

```
@@ -27,6 +27,18 @@ endif # SOC_FLASH_MCUX
 
 if HAS_MCUX_FLEXSPI
 
+config USE_DEDICATED_FLEXSPI_FLASH_DRIVER
+	bool "Use a dedicated flexspi flash driver"
+	help
+	  Enables choosing a flexspi flash driver.
+
+if USE_DEDICATED_FLEXSPI_FLASH_DRIVER
+
+choice FLEXSPI_FLASH_DRIVER
+	prompt "Flexspi flash driver"
+	help
```

```suggestion
	optional
	help
```

Related to the `USE_DEDICATED_FLEXSPI_FLASH_DRIVER` comment.

---

Made the changes to optional which is a better solution